### PR TITLE
Exclude virtualenvs vers w/ regressions 20.0.[0-7]

### PR DESCRIPTION
**Pull request by webknjaz**
virtualenv 20 series has had a number of regressions. This PR explicitly excludes versions with known issues that are hard to debug.

## Contribution checklist:

(also see [CONTRIBUTING.rst](../tree/master/CONTRIBUTING.rst) for details)

- [x] wrote descriptive pull request text
- [ ] added/updated test(s)
- [ ] updated/extended the documentation
- [x] added relevant [issue keyword](https://help.github.com/articles/closing-issues-using-keywords/)
      in message body
- [ ] added news fragment in [changelog folder](../tree/master/docs/changelog)
  * fragment name: `<issue number>.<type>.rst` for example (588.bugfix.rst)
  * `<type>` is must be one of `bugfix`, `feature`, `deprecation`,`breaking`, `doc`, `misc`
  * if PR has no issue: consider creating one first or change it to the PR number after creating the PR
  * "sign" fragment with "by :user:`<your username>`"
  * please use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files - by :user:`superuser`."
  * also see [examples](../tree/master/docs/changelog)
- [x] added yourself to `CONTRIBUTORS` (preserving alphabetical order)

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -43,7 +43,7 @@ install_requires =
     pluggy >= 0.12.0, <1
     py >= 1.4.17, <2
     six >= 1.14.0, <2  # required when virtualenv>=20
-    virtualenv >= 16.0.0
+    virtualenv >= 16.0.0, !=20.0.0, !=20.0.1, !=20.0.2, !=20.0.3, !=20.0.4, !=20.0.5, !=20.0.6, !=20.0.7
```
Comment (gaborbernat) [inline setup.cfg]: Why is this needed? Shouldn't install just pick the latest?

Comment (asottile) [reply setup.cfg]: additionally, though each of these have bugs they "work" for the 95-99% case and it's probably not helpful to exclude it for those already using those versions

Comment (webknjaz) [reply setup.cfg]: It'll only install latest if no existing virtualenv is present. So if you do `pip install tox` and there's `virtualenv==20.0.2` there, it'll not upgrade it because it matches the requirement.

Comment (webknjaz) [reply setup.cfg]: I can't think of a better solution atm

Comment (gaborbernat) [reply setup.cfg]: 😢 you're right

**Comment by gaborbernat** [comment]
released via https://pypi.org/project/tox/3.14.6/
